CHANGELOG.rst

```diff
@@ -1,28 +1,54 @@
 Changelog
 =========
 
-Not Yet Released
-----------------
-
-- A new infrastructure for converting ``EventList`` and ``LightCurve`` objects into Astropy ``TimeSeries``
-- A new infrastructure for converting most ``Stingray`` classes into Astropy ``Table`` objects
-- Save and load of most ``Stingray`` classes to/from many different file formats (``pickle``, ``ECSV``, ``HDF5``, ``FITS``, and all formats compatible with Astropy Table)
-- Accept input ``EventList`` in ``DynamicalPowerSpectrum``
+v1.0beta (2022-02-25)
+---------------------
+TL,DR: these things will break your code with v1.0beta:
+
+- Python version < 3.8
+- The `gtis` keyword in `pulse/pulsar.py` (it is now `gti`, without the 's')
+
+New
+^^^
+- Dropped support to Python < 3.8
+- Multi-taper periodogram, including a Lomb-Scargle implementation for non-uniformly sampled data
+- Create Count rate spectrum when calculating spectral-timing products
+- Make modulation upper limit in ``(Averaged)Powerspectrum`` work with any normalization (internally converts to Leahy for the calculation)
 - Implement Gardner-Done normalization (1 for perfect correlation, -1 for perfect anticorrelation) for ``Auto/Crosscorrelation``
-- Make modulation upper limit in ``(Averaged)Powerspectrum`` work with any normalization
-- Dropped support to Python <3.8
-- Multi-taper periodogram, including a Lomb-Scargle implementation for non-uniformly sampled data.
-- Upper limits on pulsations in periodograms and Z searches
-- New stingray.fourier module containing the basic timing products, usable on ``numpy`` arrays
-- Lots of performance improvements in the ``AveragedCrossspectrum`` and ``AveragedPowerspectrum`` classes
-- New methods in ``Crossspectrum`` and ``Powerspectrum`` to load data from specific inputs: ``from_events``, ``from_lightcurve``, ``from_time_array``, ``from_lc_list``
-    - ``from_time_array`` was also tested using memory-mapped event lists as inputs: useful in very large datasets
+- New infrastructure for converting ``EventList`` and ``LightCurve`` objects into Astropy ``TimeSeries``
+- New infrastructure for converting most ``Stingray`` classes into Astropy ``Table`` objects, ``xarray`` and ``pandas`` data frames.
```

Comment:
Do you want double-backticks around Stingray, Pandas, etc? (Here and throughout)

To make that more concrete: I would suggest you probably want double-backticks around literals (ie, things that actually appear in the code — classes, functions, parameters, etc) but not around proper nouns (names of software products, like Stingray, Astropy, Numpy, etc).